games/static/games/proto.js

```diff
@@ -456,13 +459,16 @@ class Board {
     let nextRocketLocation = this.getNextIndexOf(rocket);
     this.squareAt(rocketLocation).setCharacter(rocket);
     rocketId = setInterval(() => {
-      let rocketInFlight = this.liftRocketAt(rocketLocation);
+      if (!this.squareAt(rocketLocation).hasRocket()) {
+        this.squareAt(rocketLocation).setEmpty()
```

Comment:
is this safe? maybe the square has something else on it?
why do we need it?

Reply:
without it sometimes the interval would put an alien on top of the rocket so this kills the alien that moved into the rocket, otherwise weird things happen